sys/kern/kern_sig.c

```diff
@@ -90,23 +90,16 @@ __FBSDID("$FreeBSD$");
 #include <vm/vm.h>
 #include <vm/vm_extern.h>
 #include <vm/uma.h>
-
-#include <sys/jail.h>
-
-// TODO NWF: Audit headers
-#if __has_feature(capabilities)
-#include <cheri/cheri.h>
-#include <cheri/cheric.h>
 #ifdef CHERI_CAPREVOKE
+#include <vm/vm_cheri_revoke.h>
 #ifdef CHERI_CAPREVOKE_STATS
 #include <vm/pmap.h>
 #include <vm/vm_map.h>
 #endif
-#include <vm/vm_cheri_revoke.h>
-#include <cheri/revoke.h>
-#endif
 #endif
 
+#include <sys/jail.h>
```

Comment:
This is in the wrong place?

Reply:
I think so, but also upstream so I left it alone for now. I'll attempt to fix it upstream.

Reply:
Committed as freebsd/freebsd-src@437e1e37dfcaa645f6cf5f1ddd98bbe68b4d4462.